install/agones/README.md

```diff
@@ -0,0 +1,83 @@
+# Agones
+
+[Agones](https://agones.dev) is designed as a batteries-included, open-source, dedicated game server hosting and scaling project built on top of Kubernetes, with the flexibility you need to tailor it to the needs of your multiplayer game.
+
+## Introduction
+
+This chart install the Agones application and defines deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
```

Comment:
It makes sense to me. The document `installing_agones.md` points to this file if the user wants to install Agones using Helm.

You're right about the supported version. Currently I am doing this functionality using version `2.8.2` on the client and server. I will put this version in the document, but it would be good to try other versions, wdyt?